packages/app/src/rhdh.json

```diff
@@ -0,0 +1,4 @@
+{
+  "rhdhVersion": "1.0.0",
+  "backstageVersion": "1.22.1"
```

Comment:
could this be inherited from https://github.com/janus-idp/backstage-showcase/blob/main/backstage.json#L2 so we don't need a new config file with the same information?

Reply:
https://github.com/janus-idp/backstage-showcase/pull/927#discussion_r1468085932

Reply:
can we generate this at build time then? https://github.com/janus-idp/backstage-showcase/pull/927#discussion_r1468199818 ? 

do you plan to also collect a build timestamp or other metadata fields? or would this be something I overrride downstream so we can have "1.1.0-CI-timestamp" instead of just "1.1.0" ?